pages/express-relay/integrate-as-searcher/svm.mdx

```diff
@@ -0,0 +1,208 @@
+import { Callout, Tabs, Steps } from "nextra/components";
+
+# SVM Searcher Integration
+
+SVM Express Relay searchers fulfill opportunities representing limit orders on the [Limo]() program.
+
+<Steps>
+
+### Subscribe to New Opportunities
+
+Express Relay provides searchers with [Typescript](https://github.com/pyth-network/per/tree/8e311d3dce7a54865ff98b25e57c6af2dd984d1f/sdk/js) and [Python](https://github.com/pyth-network/per/tree/8e311d3dce7a54865ff98b25e57c6af2dd984d1f/sdk/python) SDKs to interact with Express Relay.
+Searchers can also directly fetch available opportunities via HTTP or subscribe to them via WebSocket.
+
+<Tabs items={['Typescript', 'Python', 'HTTP', 'Websocket']}>
+
+<Tabs.Tab>
+Pyth provides a Typescript SDK, which allows searchers to subscribe to opportunities:
+
+```typescript
+import { Client, Opportunity } from "@pythnetwork/express-relay-js";
+
+const handleOpporunity = async (opportunity: Opportunity) => {
+  // Implement your opportunity handler here
+};
+
+const client = new Client(
+  { baseUrl: "https://pyth-express-relay-mainnet.asymmetric.re" },
+  undefined, // Default WebSocket options
+  handleOpporunity
+);
+await client.subscribeChains(["development-solana"]);
+```
+
+</Tabs.Tab>
+<Tabs.Tab>
+Pyth provides a Python SDK, which allows searchers to subscribe to opportunities:
```

Comment:
Is this Pyth or Douro? Can you confirm that?

Reply:
this is what we've had in the past. and i think this is fine, service owned by the DAO